app/src/main/java/org/simple/clinic/facility/alertchange/AlertFacilityChangeSheet.kt

```diff
@@ -30,21 +33,47 @@ class AlertFacilityChangeSheet : BottomSheetActivity() {
   companion object {
     const val FACILITY_CHANGE = 101
     private const val CURRENT_FACILITY_NAME = "current_facility"
+    private const val CONTINUE_TO_SCREEN = "continue_to_screen"
+
+    const val EXTRA_CONTINUE_SCREEN = "extra_continue_screen"
 
     fun intent(context: Context, currentFacilityName: String): Intent {
       val intent = Intent(context, AlertFacilityChangeSheet::class.java)
       intent.putExtra(CURRENT_FACILITY_NAME, currentFacilityName)
       return intent
     }
+
+    fun intentForScreen(
+        context: Context,
+        currentFacilityName: String,
+        continueToScreen: FullScreenKey
+    ): Intent {
+      val intent = Intent(context, AlertFacilityChangeSheet::class.java)
+      intent.putExtra(CURRENT_FACILITY_NAME, currentFacilityName)
+      intent.putExtra(CONTINUE_TO_SCREEN, continueToScreen)
+      return intent
+    }
+
+    fun <T : Parcelable> readExtra(intent: Intent): T {
```

Comment:
Should we rename this to something like `readContinuationExtra` since just `Extra` is also applicable to any of the extras in the `Intent`?